.github/workflows/charts.yml

```
@@ -18,7 +18,11 @@ jobs:
         uses: helm/chart-testing-action@v2.4.0
 
       - name: Run lint
-        run: ct lint --config .github/ct.yaml --charts=helm-chart
+        run: |
```

Did we try the following and that didn't work?
```suggestion
        run: ct lint --config .github/ct.yaml --all
```